source/Symbol/SwiftASTContext.cpp

```diff
@@ -5897,27 +5920,37 @@ SwiftASTContext::GetSwiftFixedTypeInfo(void *type) {
 
 uint64_t SwiftASTContext::GetBitSize(lldb::opaque_compiler_type_t type,
                                      ExecutionContextScope *exe_scope) {
-  if (type) {
-    swift::CanType swift_can_type(GetCanonicalSwiftType(type));
-    // FIXME: Query remote mirrors for this.
-    if (swift_can_type->hasTypeParameter())
-      return GetPointerByteSize() * 8;
+  if (!type)
+    return 0;
 
-    const swift::TypeKind type_kind = swift_can_type->getKind();
-    switch (type_kind) {
-    case swift::TypeKind::LValue:
-    case swift::TypeKind::UnboundGeneric:
-    case swift::TypeKind::GenericFunction:
-    case swift::TypeKind::Function:
+  swift::CanType swift_can_type(GetCanonicalSwiftType(type));
+  if (swift_can_type->hasTypeParameter()) {
+    if (!exe_scope)
+      // FIXME: Return an error instead.
       return GetPointerByteSize() * 8;
-    default:
-      break;
-    }
-    const swift::irgen::FixedTypeInfo *fixed_type_info =
-        GetSwiftFixedTypeInfo(type);
-    if (fixed_type_info)
-      return fixed_type_info->getFixedSize().getValue() * 8;
+    ExecutionContext exe_ctx;
+    exe_scope->CalculateExecutionContext(exe_ctx);
+    auto swift_scratch_ctx_lock = SwiftASTContextLock(&exe_ctx);
+    CompilerType bound_type = BindAllArchetypes({this, type}, exe_scope);
+    // Note thay the bound type may be in a different AST context.
+    return bound_type.GetBitSize(exe_scope);
   }
+
+  const swift::TypeKind type_kind = swift_can_type->getKind();
```

Comment:
This part makes no sense. LValues, UnboundGenerics and GenericFunctions are not real types -- nothing in memory has that type. And Functions are not 8 pointers in size.

Reply:
That's part of the original code I didn't touch. I'll see if I can just remove those cases.

Reply:
Removing those cases breaks:
    lldb-Suite :: lang/swift/variables/func/TestFunctionVariables.py
    lldb-Suite :: lang/swift/variables/inout/TestInOutVariables.py

It looks like at least LLDB itself is still generating LValue types.

Reply:
We should really fix this. Can you file a rdar? (I'll be happy to take a stab at it).

Reply:
How do they break?

Reply:
Looks like I already filed rdar://41608567, and it was closed. I guess there were two overloads of this function. Sigh...